.gitignore

```diff
@@ -148,3 +148,6 @@ dist
 
 # Lighthouse performance testing
 .lighthouseci/ 
+
+# vs code
+/.vscode/
```

Comment:
Please add a newline to the end of the file, this can cause problems on some system configurations and GitHub doesn't seem to be a fan of it either.